jekyll/_cci2/configuration-reference.md

```diff
@@ -404,42 +404,94 @@ A job that was not executed due to configured rules will show up in the list of
-small       | 1 | 2GB
-medium (default) | 2 | 4GB
-medium+     | 3 | 6GB
-large       | 4 | 8GB
-xlarge      | 8 | 16GB
-2XL         | 16 | 32GB
-2XL+        | 20 | 40GB
-1GPU        | 16 | 122GiB
-2GPU        | 32 | 244GiB
-4GPU        | 64 | 488GiB
-{: class="table table-striped"}
+**Note:** The `resource_class` feature is automatically enabled on Performance Plan. If you are on a container or unpaid plan you will need to [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) to speak with a CircleCI Sales representative about enabling this feature on your account.
+
+It is possible to configure CPU and RAM resources for each job. If `resource_class` is not specified, or an invalid class is specified, the default `resource_class: medium` will be used. Different resource classes are available for different executors, as described in the tables below. 
 
-**Note:** Approval from the CircleCI support team is required to gain access to the 2XL, 2XL+, and GPU resource classes.
+##### Docker Executor
 
-Below is an example of specifying the `large` `resource_class`.
+Class            | vCPUs | RAM
+-----------------|-------|-----
+small            | 1     | 2GB
+medium (default) | 2     | 4GB
+medium+          | 3     | 6GB
+large            | 4     | 8GB
+xlarge           | 8     | 16GB
+2xlarge\*        | 16    | 32GB
+2xlarge+\*       | 20    | 40GB
+{: class="table table-striped"}
 
+###### Example Usage
 ```yaml
 jobs:
   build:
     docker:
       - image: buildpack-deps:trusty
-    environment:
-      FOO: bar
-    parallelism: 3
+    resource_class: xlarge
+    steps:
+      ... // other config
+```
+
+##### Machine Executor (Linux)
+
+Class            | vCPUs | RAM
+-----------------|-------|-------
+medium (default) | 2     | 7.5GB
+large            | 4     | 15GB
+1GPU\*           | 16    | 122GiB
+2GPU\*           | 32    | 244GiB
+4GPU\*           | 64    | 488GiB
+{: class="table table-striped"}
+
+###### Example Usage
+```yaml
+jobs:
+  build:
+    machine: true
+    resource_class: 1GPU
+    steps:
+      ... // other config
+```
+
+##### macOS Executor
+
+Class            | vCPUs | RAM
+-----------------|-------|-----
+medium (default) | 4     | 8GB
+large\*          | 8     | 16GB
+{: class="table table-striped"}
+
+###### Example Usage
+```yaml
+jobs:
+  build:
+    macos:
+      xcode: "11.0.0"
     resource_class: large
     steps:
-      - run: make test
-      - run: make
+      ... // other config
 ```
 
+##### Windows Executor
+
+Class             | vCPUs | RAM
+------------------|-------|-----
+medium (default)  | 4     | 15GB
+
+###### Example Usage
+```yaml
+jobs:
+  build:
+    machine:
+      image: windows-server-2019-vs2019:201908-06
+    resource_class: windows.medium
+    shell: powershell.exe -ExecutionPolicy Bypass
+    steps:
+      ... // other config
+```
+
+\*_Requires review. Open a support ticket [here](https://support.circleci.com/hc/en-us/requests/new)._
```

Comment:
There's a new process to request access now, and this is the direct link: https://docs.google.com/forms/d/11ugpC64aO9P1DrI0VE29k0l_WD14qiGE-Cao7i5KYPA/edit

Reply:
Hi @rosaliebud , I thought that form is for one of us to fill out on behalf of a customer and should not be put into public documentation.

Is that a correct understanding?